ui/packages/catalog-ui-search/src/main/webapp/js/widgets/openlayers.polygon.js

```diff
@@ -11,252 +11,251 @@
  **/
 /*global define, window*/
 
-define([
-        'marionette',
-        'backbone',
-        'openlayers',
-        'underscore',
-        'properties',
-        'wreqr',
-        'maptype',
-        './notification.view',
-        'js/ShapeUtils',
-        './drawing.controller',
-        '../OpenLayersGeometryUtils'
-    ],
-    function(Marionette, Backbone, ol, _, properties, wreqr, maptype, NotificationView, ShapeUtils, DrawingController, olUtils) {
-        "use strict";
-
-        var Draw = {};
-
-        Draw.PolygonModel = Backbone.Model.extend({
-            defaults: {
-                north: undefined,
-                east: undefined,
-                west: undefined,
-                south: undefined
-            }
+const Marionette = require('marionette');
+const Backbone = require('backbone');
+const ol = require('openlayers');
+const _ = require('underscore');
+const properties = require('properties');
+const wreqr = require('wreqr');
+const NotificationView = require('./notification.view');
+const ShapeUtils = require('js/ShapeUtils');
+const Turf = require('@turf/turf');
+const DrawingController = require('./drawing.controller');
+const olUtils = require('../OpenLayersGeometryUtils');
+const DistanceUtils = require('js/DistanceUtils');
+
+const translateFromOpenlayersCoordinates = (coords) => {
+    return coords.map(value => value.map(point => {
+        const mappedPoint = ol.proj.transform([point[0], point[1]], properties.projection, 'EPSG:4326');
+        if (mappedPoint[1] > 90) {
+            mappedPoint[1] = 89.9;
```

Comment:
❓ Why do we use 89.9 instead of 90?

Reply:
Legacy code that I copied over, honestly not sure